### PR TITLE
Format ISO 8601 for Calendar.Time

### PR DESCRIPTION
**Pull request by Fabi755**
Add function for get ISO 8601 of `Calendar.Time`.
Add function for parse ISO 8601 of `Calendar.Time`.
Add missing `Calendar.DateTime.Format.iso_8601`.
Fix naming of ISO 8601 functions.

### CODE REVIEW

**Comment by lau** [comment]
Thanks for the time parser.
Could you undo the changes to the `iso8601` function names?

File: lib/calendar/date_time/format.ex [hunk 1]
```diff
@@ -64,6 +64,23 @@ defmodule Calendar.DateTime.Format do
...
   @doc """
+  Format as ISO 8601 extended
+
+  # Examples
+
+      iex> Calendar.DateTime.from_erl!({{2014, 9, 26}, {20, 10, 20}}, "Etc/UTC",5) |> Calendar.DateTime.Format.iso_8601
+      "2014-09-26T20:10:20Z"
+      iex> Calendar.DateTime.from_erl!({{2014, 9, 26}, {17, 10, 20}}, "America/Montevideo",5) |> Calendar.DateTime.Format.iso_8601
+      "2014-09-26T17:10:20-0300"
+  """
```
Comment (lau) [inline lib/calendar/date_time/format.ex:68]: Maybe we could make an alias to `rfc3339` instead.

**Comment by Fabi755** [comment]
Yes, I will undo the name fixes.

I think you are right. An alias for rfc3339 is the better way.

**Comment by lau** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**